src/saveload/afterload.cpp

```diff
@@ -2486,6 +2486,12 @@ bool AfterLoadGame()
 		for (Depot *d : Depot::Iterate()) d->build_date = TimerGameCalendar::date;
 	}
 
+	if (IsSavegameVersionBefore(SLV_145)) {
```

Comment:
There's a `!IsSavegameVersionBefore(SLV_145)` block line 2831, cannot they be merged?

Reply:
Ah no, they are at different levels.